.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.1
```

Comment:
This file has been generated by `rbenv` and I think it should be included in repo